discord.photography/Portfolios.html

```diff
@@ -21,14 +21,13 @@
 
 	<div class="main">
 		<h4>
```

Comment:
Its not really good wrapping all links in a single h4 - accessibility wise its also not a h4, maybe h1 if at all and then style it as h4. Yet would rather not use it at all and just style the hrefs. Since this is out of scope of this PR, we may need to change that in a different PR